empmgmt-servlet/src/main/java/com/neha/empmgmt/dao/impl/EmployeeDaoImpl.java

```diff
@@ -14,58 +16,172 @@
 
 	@Override
 	public Employee findById(int id) {
+		Employee employee = new Employee();
 		String sql = "Select * from employee where id = ?";
 		PreparedStatement stmt = null;

```

Comment:
Please format this class and any other that we might have missed. Otherwise it looks okay to me. Please make the changes and commit again. A new commit will just update the same pull request and then I will approve that.